src/context/delta.rs

```diff
@@ -89,7 +89,7 @@ fn temp_partition_file_writer(
 
 /// Execute a plan and upload the results to object storage as Parquet files, indexing them.
 /// Partially taken from DataFusion's plan_to_parquet with some additions (file stats, using a DiskManager)
-pub async fn plan_to_object_store(
+pub async fn plan_to_delta_data(
```

Comment:
Maybe also
```suggestion
pub async fn plan_to_delta_adds(
```
to make it more concrete